src/main/java/maxsuperman/addons/roller/modules/VillagerRoller.java

```diff
@@ -735,16 +755,19 @@ private void onTick(TickEvent.Pre event) {
                     return;
                 }
             }
-            if (mc.world.getBlockState(rollingBlockPos) == Blocks.AIR.getDefaultState()) {
-                info("Lectern placement reverted by server (AC?)");
-                currentState = State.RollingPlacingBlock;
-                return;
-            }
+            if(!baritonePlacing.get()) {
+                //This can (most of the time) never happen with baritone
+                if (mc.world.getBlockState(rollingBlockPos) == Blocks.AIR.getDefaultState()) {
+                    info("Lectern placement reverted by server (AC?)");
+                    currentState = State.RollingPlacingBlock;
+                    return;
+                }
             if (!mc.world.getBlockState(rollingBlockPos).isOf(Blocks.LECTERN)) {
                 info("Placed wrong block?!");
                 currentState = State.RollingBreakingBlock;
                 return;
             }
+            }
```

Comment:
It still can actually, better inform user about it properly.
Also what happened to indentation here?